### PR TITLE
Add a story book entry for all 'atoms'

### PR DESCRIPTION
**Pull request by martinfrances107**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/186

## Screenshot / UI changes

None

## Testing instructions

start storybook server with 

yarn storybook 

Observe the new storybook entry for the OpModalButton in the left hand column.

Please describe with a few words how to reproduce the problem / test the feature

I want to provide a example/reference implementation of the opModalButton component.

### CODE REVIEW

File: src/components/02_atoms/Widgets/Widgets.stories.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { storiesOf } from '@storybook/react';
+
+import BooleanCheckbox from './BooleanCheckbox';
+import FileUploadWidget from './FileUploadWidget';
+import NumberTextfield from './NumberTextfield';
+import OptionsSelect from './OptionsSelect';
+import StringTextfield from './StringTextfield';
+import TimestampDatetime from './TimestampDatetime';
+
+const mockOnChange = e => {
```
[the remark this answers is not in the document]
Comment (martinfrances107) [reply src/components/02_atoms/Widgets/Widgets.stories.js]: Sure I will clean this up ... but I won't have the time  before Mondays meeting :)

[the remark this answers is not in the document]
Comment (dawehner) [reply src/components/02_atoms/Widgets/Widgets.stories.js]: No worries at all, take your time

**Comment by dawehner** [comment]
@martinfrances107 do you want to address the one point I made in the review? I think otherwise this PR looks great at the moment.

**Comment by martinfrances107** [comment]
Ok I have finally fix this up.

There is a known issue between .addWithJSX and actions ...

But there is a workaround by following a convention with toString() call.

Sorry for the delay ... i just did not have the spare time.

File: src/components/02_atoms/FileUpload/FileUpload.stories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { storiesOf } from '@storybook/react';
+
+import FileUpload from './FileUpload';
+
+storiesOf('FileUpload', module).addWithJSX('Default', () => (
+  <FileUpload
+    entityTypeId="node"
+    bundle="Article"
+    fieldName="upload"
+    onFileUpload={() => {}}
```
[the remark this answers is not in the document]
Comment (martinfrances107) [reply src/components/02_atoms/FileUpload/FileUpload.stories.js]: Fixed.

File: src/components/02_atoms/OpsModalButton/OpsModalButton.stories.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <TableRow>
+          <TableCell>Push button to see the modal pop up.</TableCell>
+          <TableCell>
+            <OpsModalButton
+              aria-label="edit"
+              title="Are you sure that you want to edit this element?"
+              text="This action cannot be undone."
+              cancelText="Cancel"
+              confirmText="Edit"
+              enterAction={() => {}}
```
[the remark this answers is not in the document]
Comment (martinfrances107) [reply src/components/02_atoms/OpsModalButton/OpsModalButton.stories.js]: Done.

File: src/components/02_atoms/Widgets/Widgets.stories.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (martinfrances107) [reply src/components/02_atoms/Widgets/Widgets.stories.js]: It is OK now.

File: src/components/02_atoms/FileUpload/FileUpload.stories.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+/**
+ * There is a known issue with addWithJSX and action() calls.
+ *
+ *  https://github.com/storybooks/addon-jsx/issues/30
+ *
+ * The get both the logger to function while still pretty printing sample code
+ * in the JSX tab a toSting() method must be provided.
+ */
+const onFileUploadAction = action('onFileUpload');
```
Comment (dawehner) [inline src/components/02_atoms/FileUpload/FileUpload.stories.js]: Interesting research!

**Comment by dawehner** [comment]
Thank you for addressing the feedback @martinfrances107 !

**Pull request merged**